source/includes/_reporting-entities.md

```diff
@@ -1,4 +1,4 @@
-# Reporting Entities
+# Reporting Entities PLACEHOLDER
 
 <aside class="warning">
   This set of endpoints replaces the previous "Survey Responses" endpoints. Only
```

Comment:
This warning we can remove since that endpoint has been long gone and nobody uses it anymore.